<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metropark Pulse</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/ico/PNG/pulse.png</normaloff>:/ico/PNG/pulse.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {background-image: url(:/ico/PNG/64/Gnome-Emblem-Documents-64.png);
background-repeat:no-repeat;
background-attachment:fixed;
background-position:bottom right; 
text-align:left;
}

QHeaderView::section
{
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #A60000, stop: 0.5 #BF3030,
                                       stop: 0.6 #BF3030, stop:1 #A60000);
     color: white;
     border: 1px solid #6c6c6c;
	 padding: 3px;
	font-weight: bold;
}</string>
      </property>
      <property name="title">
       <string>Hot Tickets</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_tickets_error">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Loading...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table_tickets">
         <property name="styleSheet">
          <string notr="true">QTableWidget {background-image: url(:/ico/PNG/64/Gnome-Emblem-Documents-64.png);
background-repeat:no-repeat;
background-attachment:fixed;
background-position:bottom right; 
text-align:left;
}

QHeaderView::section
{
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #A60000, stop: 0.5 #BF3030,
                                       stop: 0.6 #BF3030, stop:1 #A60000);
     color: white;
     border: 1px solid #6c6c6c;
	 padding: 3px;
	font-weight: bold;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Priority</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ticket</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Owned By</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Working</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Worked Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Customer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Ticket Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>System</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="action_Quit"/>
    <addaction name="separator"/>
    <addaction name="actionReload_Network"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionNetwork_Browser"/>
    <addaction name="actionTechs"/>
    <addaction name="separator"/>
    <addaction name="actionTheatre_View"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QDockWidget" name="dock_network_browser">
   <property name="features">
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Network</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="indentation">
           <number>10</number>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_zone_error">
          <property name="text">
           <string>Loading...</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dock_techs">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>188</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QHeaderView::section
{
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #619161, stop: 0.5 #508050,
                                       stop: 0.6 #437343, stop:1 #659565);
     color: white;
     border: 1px solid #6c6c6c;
    padding: 3px;
	font-weight: bold;
}</string>
       </property>
       <property name="title">
        <string>Who's Working What</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableWidget" name="table_techs">
          <property name="styleSheet">
           <string notr="true">QTableWidget {background-image: url(:/ico/PNG/64/Gnome-System-Users-64.png);
background-repeat:no-repeat;
background-attachment:fixed;
background-position:bottom right; 
text-align:left;}
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Tech</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Priority</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time Worked</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Customer</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ticket Working</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_techs_error">
          <property name="text">
           <string>Loading...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ico/PNG/48/Gnome-Preferences-System-48.png</normaloff>:/ico/PNG/48/Gnome-Preferences-System-48.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ico/PNG/32/Gnome-System-Shutdown-32.png</normaloff>:/ico/PNG/32/Gnome-System-Shutdown-32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionReload_Network">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ico/PNG/64/Gnome-Preferences-Desktop-Remote-Desktop-64.png</normaloff>:/ico/PNG/64/Gnome-Preferences-Desktop-Remote-Desktop-64.png</iconset>
   </property>
   <property name="text">
    <string>Reload Network</string>
   </property>
  </action>
  <action name="actionFont_Size">
   <property name="text">
    <string>Font Size</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ico/PNG/pulse.png</normaloff>:/ico/PNG/pulse.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionNetwork_Browser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ico/PNG/64/Gnome-Preferences-Desktop-Remote-Desktop-64.png</normaloff>:/ico/PNG/64/Gnome-Preferences-Desktop-Remote-Desktop-64.png</iconset>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </action>
  <action name="actionTechs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ico/PNG/64/Gnome-System-Users-64.png</normaloff>:/ico/PNG/64/Gnome-System-Users-64.png</iconset>
   </property>
   <property name="text">
    <string>Who's Working What</string>
   </property>
  </action>
  <action name="actionTheatre_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Theatre View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>Quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload_Network</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>LoadZone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNetwork_Browser</sender>
   <signal>toggled(bool)</signal>
   <receiver>dock_network_browser</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTechs</sender>
   <signal>toggled(bool)</signal>
   <receiver>dock_techs</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>766</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTheatre_View</sender>
   <signal>triggered(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowTheatre(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LoadTickets()</slot>
  <slot>ShowSettings()</slot>
  <slot>LoadZone()</slot>
  <slot>Quit()</slot>
  <slot>ShowAbout()</slot>
  <slot>NetworkVisibilityChanged(bool)</slot>
  <slot>ShowTheatre(bool)</slot>
 </slots>
</ui>
